docs/publishing/sections/7_ckan.md

```diff
@@ -0,0 +1,65 @@
+(publishing-ckan)=
+# Publishing data to California Open Data aka CKAN
+
+NOTE: Only non-spatial data should be directly published to CKAN. Spatial data
+(i.e. data keyed by location in some manner) has to go through the Caltrans
+geoportal and is subsequently synced to CKAN.
+
+## What is the California Open Data Portal?
+
+The state of California hosts its own instance of CKAN, called the [California Open Data Portal](https://data.ca.gov/).
+[CKAN](https://ckan.org/) is an open-source data management tool; in other words,
+it allows organizations such as Caltrans to host data so that users can search
+for and download data sets that might be useful for analysis or research. Among
+other agencies, Caltrans publishes [many data sets](https://data.ca.gov/organization/caltrans) to CKAN.
+Data is generally published as flat files (typically CSV) alongside required
+metadata and a data dictionary.
+
+## Cal-ITP data sets
+* [Cal-ITP GTFS Schedule Data](https://data.ca.gov/dataset/cal-itp-gtfs-ingest-pipeline-dataset)
+
+## General process
+### Develop data models
+Generally, data models should be built in dbt/BigQuery if possible. For example,
+we have [latest-only GTFS schedule models](https://github.com/cal-itp/data-infra/tree/main/warehouse/models/gtfs_schedule_latest_only)
+we can use to update and expand the existing [CKAN dataset](https://data.ca.gov/dataset/cal-itp-gtfs-ingest-pipeline-dataset)
+[latest-only agency](https://dbt-docs.calitp.org/#!/model/model.calitp_warehouse.agency)
+
+### Document data
+California Open Data requires two documentation files for published datasets.
+1. `metadata.csv` - one row per resource (i.e. file) to be published
+2. `dictionary.csv` - one row per column across all resources
+
+If you are using dbt exposure-based data publishing, you can automatically generate
+these two files using the main `publish.py` script (specifically the `generate-exposure-documentation`
+subcommand). The documentation from the dbt models' corresponding YAML will be
+converted into appropriate CSVs and written out locally.
```

Comment:
can you add a literal example call for this command? and specify where it would be run.... is it getting a local manifest or are you providing a prod manifest that it uses? (I may be misunderstanding something below but I thought it needs a manifest?)